### PR TITLE
Umar/load new course from GitHub

### PR DESCRIPTION
**Pull request by umarmughal824**
#### Pre-Flight checklist

- [x] Screenshots and design review for any changes that affect layout or styling
  - [x] Desktop screenshots
  - [x] Tag @ferdi or @pdpinch for review
- [x] Testing
  - [x] Code is tested
  - [x] Changes have been manually tested

#### What are the relevant tickets?
#4 

#### What's this PR do?
Load new course from Github (Git Import)

#### How should this be manually tested?
Just visit the page

#### Where should the reviewer start?
(Optional)

#### Any background context you want to provide?
(Optional)

#### Screenshots (if appropriate)
Screenshot 2021-03-26 at 16 25 33
Screenshot 2021-03-26 at 16 25 43
Screenshot 2021-03-26 at 16 26 00
Screenshot 2021-03-26 at 16 26 12
Screenshot 2021-03-26 at 17 40 09
Screenshot 2021-03-26 at 16 36 56
Screenshot 2021-03-26 at 16 37 21

### CODE REVIEW

File: edx_sysadmin/views.py [hunk 1]
```diff
@@ -149,6 +157,118 @@ def post(self, request, *args, **kwargs):  # pylint: disable=unused-argument
...
 
 
+class GitImport(SysadminDashboardView):
+    """
+    This provide the view to load new course from github
+    """
+    
+    template_name = 'edx_sysadmin/gitimport.html'
+
+    def git_info_for_course(self, cdir):
```
Comment (ziafazal) [inline edx_sysadmin/views.py:167]: @umarmughal824 where is this method being used?

Comment (umarmughal824) [reply edx_sysadmin/views.py:167]: @ziafazal bhai, it was used in showing the table of the courses but now we are not using it anymore as per @pdpinch suggestion. So removing that function for now.

**Review by ziafazal — APPROVED** [review]
👍

**Comment by umarmughal824** [comment]
@pdpinch could you please take a look at the attached screenshots.

**Pull request merged**